test/e2e/flask/user-operations.spec.ts

```diff
@@ -253,12 +254,7 @@ describe('User Operations', function () {
     await withAccountSnap(
       { title: this.test?.fullTitle() },
       async (driver, bundlerServer) => {
-        await sendTransaction(
-          driver,
-          GANACHE_ACCOUNT,
-          convertETHToHexGwei(1),
```

Comment:
why is the `convertETHToHexGwei` call no longer needed?

Reply:
Oh it was rounding down to 0!

Reply:
Gotcha!